src/index.css

```
@@ -1,3 +1,13 @@
+@import "tailwindcss";
+
+@tailwind base;
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)